filebeat/filebeat.template-es2x.json

```diff
@@ -40,95 +43,123 @@
                 "body_sent": {
                   "properties": {
                     "bytes": {
+                      "doc_values": true,
+                      "index": true,
```

Comment:
Hmm, I think `"index": true` is not valid for 2x. It should be `"index": "analyzed"`, or simply leave the default.